tests/integrations_app/flagship/test_flashy.py

```diff
@@ -0,0 +1,79 @@
+import os
+from time import sleep
+
+import pytest
+from integrations_app.flagship import _PATH_INTEGRATIONS_DIR
+
+from lightning_app.testing.testing import run_app_in_cloud
+from lightning_app.utilities.imports import _is_playwright_available
+
+if _is_playwright_available():
+    import playwright
+    from playwright.sync_api import expect, Page
+
+
+# TODO: when this function is moved to the app itself we can just import it, so to keep better aligned
```

Comment:
@ethanwharris what do you think about this eventually set it so that each app shall have defined validation that can be reused via importing the package (not a hacky import function from test file)